addon/components/cs-timetree.js

```diff
@@ -270,13 +256,11 @@ Ember.Timetree.TimetreeView = Ember.View.extend({
     var tree = this.get('tree'),
         nodes = tree.nodes(rootNode).slice(1); // Skip root node
 
-    var width = this.get('_width'),
```

Comment:
This and the other variable declarations I removed were unused, so JSHint complains.